src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -1257,7 +1267,10 @@ public void HttpWebRequest_EndGetRequestStreamContext_ExpectedValue(Uri remoteSe
             await LoopbackServer.CreateServerAsync((server, url) =>
             {
                 HttpWebRequest request = WebRequest.CreateHttp(url);
-                request.BeginGetResponse(null, null);
+                RequestState state = new RequestState();
+                state.Request = request;
+
+                request.BeginGetResponse(ResponseCallback, state);
```

Comment:
This is changing what the test was trying to test... see the name of the method.